README.md

```diff
@@ -13,10 +13,57 @@ npm install cf-buttons
 npm install cf-expandables
 ```
 
+Want some boilerplate?
+
+```sh
+npm install -g yo generator-cf
+mkdir my-new-project && cd $_
+yo cf
+```
+
 | buttons | core | expandables | forms | grid | icons | layout | pagination | typography |
 |:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|
 | [![npm](https://img.shields.io/npm/v/cf-buttons.svg?style=flat-square)](https://www.npmjs.com/package/cf-buttons)  | [![npm](https://img.shields.io/npm/v/cf-core.svg?style=flat-square)](https://www.npmjs.com/package/cf-core)  | [![npm](https://img.shields.io/npm/v/cf-expandables.svg?style=flat-square)](https://www.npmjs.com/package/cf-expandables)  | [![npm](https://img.shields.io/npm/v/cf-forms.svg?style=flat-square)](https://www.npmjs.com/package/cf-forms)  | [![npm](https://img.shields.io/npm/v/cf-grid.svg?style=flat-square)](https://www.npmjs.com/package/cf-grid)  | [![npm](https://img.shields.io/npm/v/cf-icons.svg?style=flat-square)](https://www.npmjs.com/package/cf-icons)  | [![npm](https://img.shields.io/npm/v/cf-layout.svg?style=flat-square)](https://www.npmjs.com/package/cf-layout)  | [![npm](https://img.shields.io/npm/v/cf-pagination.svg?style=flat-square)](https://www.npmjs.com/package/cf-pagination)  | [![npm](https://img.shields.io/npm/v/cf-typography.svg?style=flat-square)](https://www.npmjs.com/package/cf-typography)  |
 
+## Contributing
+
+### Building components locally
+
+```sh
+git clone git@github.com:cfpb/capital-framework.git
+cd capital-framework
+npm install
+npm run build
```

Comment:
`npm run build` runs `gulp build` [behind the scenes](https://github.com/cfpb/capital-framework/blob/canary/package.json#L27). Right now they're the same thing but it's good practice to tell people to use the npm script because in the future we might add another build step before or after `gulp build`. Or, who knows, we might move back to grunt. This way the build command stays the same.